### PR TITLE
K8SPG-597 Add all images from 2.3.1

### PR DESCRIPTION
**Pull request by ptankov**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.2.4.0.pg-operator.json [hunk 1]
```diff
@@ -34,6 +34,37 @@
...
+          "15.5": {
+            "image_path": "percona/percona-postgresql-operator:2.3.1-ppg15-postgres",
+            "image_hash": "a8ef34191c6d29f93f4a9fdde3f97f89284d67726719c94f1f7f14bd2312677e",
+            "status": "available",
+            "critical": false
+          },
+          "16.1": {
+            "image_path": "percona/percona-postgresql-operator:2.3.1-ppg16-postgres",
+            "image_hash": "2f329755dda215e233512c6e453a850bfc7505a7dfceb3a6ed64b8b84e532c15",
+            "status": "recommended",
```
Comment (hors) [inline sources/operator.2.4.0.pg-operator.json:66]:
```suggestion
            "status": "available",
```

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by tplavcic — APPROVED** [review]

**Pull request merged**